### ISSUE TITLE
Don't store rdf-toolkit.jar in repository - setup.cmd should download it

### ISSUE THREAD

**Issue by rjyounes** [semanticarts/gist#1050]

**Comment by rjyounes** [comment]
If we do this, the version number to get should be specified, since there are dependencies on version in the options given to the serializer.

**Comment by sa-bpelakh** [comment]
> If we do this, the version number to get should be specified, since there are dependencies on version in the options given to the serializer.

Agreed, we should have a version-specific URI to download. What is the rationale for this change? We wouldn't reduce the size of the cloned repo unless we retroactively removed it - otherwise it would still be in the `.git`, but just not seen.

**Comment by rjyounes** [comment]
@sa-bpelakh Rationale: I thought it was a general best practice not to store executable files in a git repository. If you disagree, feel free to close this issue. I'm not committed to it.
